programming-extensions/programming-extension-dataspaces/src/main/java/org/objectweb/proactive/extensions/dataspaces/vfs/VFSNodeScratchSpaceImpl.java

```diff
@@ -117,17 +118,21 @@ private AppScratchSpaceImpl(final long appId) throws FileSystemException {
 
         public void close() throws FileSystemException {
             logger.debug("Closing application scratch space");
+
             try {
+                final int filesNumber = spaceFile.delete(Selectors.SELECT_ALL);
+                logger.debug("Deleted " + filesNumber + " files in scratch application directory");
+            } catch (org.apache.commons.vfs2.FileSystemException e) {
+                logger.warn("Could not delete " + spaceFile, e);
+            } finally {
+                // just a hint
```

Comment:
I have updated the original message. It is related to the close operation from VFS  that does not really close the file but is just a hint to the implementation that it can release any resources associated with the file.